src/main/resources/org/jenkinsci/plugins/GithubSecurityRealm/config.jelly

```diff
@@ -16,5 +16,8 @@
 		<f:entry title="Client Secret" field="clientSecret" help="/plugin/github-oauth/help/realm/client-secret-help.html">
 			<f:textbox />
 		</f:entry>
+		<f:entry title="OAuth Scope(s)" field="oauthScopes" help="/plugin/github-oauth/help/realm/oauth-scopes-help.html">
+			<f:textbox default="repo,read:org" />
```

Comment:
There are already over a dozen oauth scopes and, hopefully, the repo scope will get split up it separate read/write and public/private scopes.  I feel from a UI perspective that would be too many check boxes.  I'd also like to not have to get a PR merged / new release made to adapt to GH scope modifications.